tests/models/owlvit/test_modeling_owlvit.py

```diff
@@ -515,11 +515,14 @@ def test_model_from_pretrained(self):
 
 
 class OwlViTForObjectDetectionTester:
-    def __init__(self, parent, is_training=True):
+    def __init__(self, parent, is_training=True, query_batch_size=12, query_image_size=32, query_num_channels=3):
```

Comment:
It might be better to use a smaller query_batch_size and smaller-than-target-image `query_image_size` for the query_image tests:
```suggestion
    def __init__(self, parent, is_training=True, query_batch_size=2, query_image_size=16, query_num_channels=3):
```